primer/brand
# Added Instagram Logo

## Summary

Added Instagram Logo to MinimalFooter Social Links

## List of notable changes:

- **added** Instagram logo to MinimalFooter social links

## What should reviewers focus on?

- Making sure the logo is the right size and color.

## Steps to test:

1. Open the MinimalFooter component in CI-deployed preview environment
2. Go to MinimalFooter story in Storybook
3. Verify that Instagram social link behaves as described in the following issue.

## Supporting resources (related issues, external links, etc):

- closes https://github.com/primer/brand/issues/266

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">



 </td>
<td valign="top">



</td>
</tr>
</table>
